src/composables/useFlawModel.ts

```diff
@@ -239,3 +240,15 @@ function flawErrors(flaw: ZodFlawType) {
 
   return mirroredFlaw;
 }
+
+export async function refreshFlawAffectTrackers(flaw: Ref<ZodFlawType>) {
+  const freshFlaw = await getFlaw(flaw.value.uuid);
```

Comment:
Should we add some kind of loader to this action?

Reply:
Hrm, maybe we should the delay should be long just to grab the flaw, but that might be best to trigger a loading state